### ISSUE TITLE
Consider more fine-grained control over the "Content" step

### ISSUE THREAD

**Issue by JiveDig** [richtabor/MerlinWP#123]
Looking at https://github.com/richtabor/MerlinWP/issues/1 got me thinking...

The same way we have the recommended plugin options, it would be nice to "choose" how much content to import.

- Posts, Pages, Custom Post Types
- Menus (though this may rely on the posts actually being there?)
- Images

This would be quite flexible for times where people just want demo posts or menus setup without requiring all the media to be downloaded.

**Comment by JiveDig** [comment]
![screen_shot_2018-10-01_at_4_57_23_pm](https://user-images.githubusercontent.com/540990/46315445-75195f80-c59b-11e8-98c6-16154b3f93d2.jpg)

**Comment by JiveDig** [comment]
Or just leave it “Content” and add “Images” and “Menus.

**Comment by contempoinc** [comment]
I think breaking it into "Content", "Images" & "Menus" would be the best route, unless its just as easy to add "Posts", "Pages" & "Custom Post Types" that'd be ideal for those people with garbage servers.

**Comment by JiveDig** [comment]
Yeah Content, Images, Menus is fine. I think it gets too crazy to split up post types at this point.

**Comment by richtabor** [comment]
I personally don't think content and images should be separated. A user probably would not want just images, or just content — or if they do, there's a high probability that they'll not be satisfied with the result of running Merlin WP. 

I do think menus should be supported though (re #1) — either in a separate checkbox, or just automagically. 🧙‍♂️

**Comment by JiveDig** [comment]
I'm not hard-sold on images being separate. It's just that sometimes images make the import take waaayyy longer (30 seconds vs 3-5 minutes in my testing). If someone is already comfortable with the theme but just wants to get some posts/pages imported and setup but they already know they are swapping out the images, then that checkbox could be useful.

But.... I agree, that may be the 20% in the 80/20 rule.

**Comment by contempoinc** [comment]
Honestly, think we need a uncheck images option for all those people out there with slow ass shared servers, its one of the main things I deal with on a daily basis.
